### DISCUSSION TITLE
Irrelevant parameters `dates`, `freq` in `statsmodels`-based models

### DISCUSSION THREAD

**Discussion by d-a-bunin** [etna-team/etna#315] [Improvements]
## Description

Currently in `statsmodels`-based models we have parameters like `freq`, `dates` which seems irrelevant and could break something.

Affected models:
- `SARIMAXModel`
- `HoltWintersModel`

The problem with `freq` is that it seems to be in conflict with `ts.freq`. The problem with `dates` is that it seems to be in conflict with `ts.index`.

## Thoughts

Possible options:
- Remove them from our models.
- Add deprecation warning on using them.
- Make them not do anything and add warning about it.

## Related

**Comment by d-a-bunin** [comment]
Created issue #588.

**Comment by egoriyaa** [comment]
Solved in #590
